File: src/main/java/no/nav/foreldrepenger/boot/conditionals/OnDevOrLocalCondition.java
package no.nav.foreldrepenger.boot.conditionals;

import static no.nav.foreldrepenger.boot.conditionals.Cluster.notProdClusters;

import org.springframework.core.type.AnnotatedTypeMetadata;

public class OnDevOrLocalCondition extends OnClusterCondition {

    @Override
    protected Cluster[] clusters(AnnotatedTypeMetadata md) {
        return notProdClusters();
    }
}

File: src/main/kotlin/no/nav/boot/conditionals/OnDevOrLocalCondition.kt
package no.nav.boot.conditionals

import no.nav.boot.conditionals.Cluster.Companion.notProdClusters
import org.springframework.core.type.AnnotatedTypeMetadata

class OnDevOrLocalCondition : OnClusterCondition() {
    override fun clusters(md: AnnotatedTypeMetadata) = notProdClusters()
}